interpreter/cling/lib/Interpreter/DynamicLibraryManager.cpp

```diff
@@ -268,4 +241,34 @@ namespace cling {
   void DynamicLibraryManager::ExposeHiddenSharedLibrarySymbols(void* handle) {
     llvm::sys::DynamicLibrary::addPermanentLibrary(const_cast<void*>(handle));
   }
+
```

Comment:
Why did you decide to move the function? That's a lot of diff for likely not much change - hard to tell due to the move... Can you leave it where it is, and just adjust the signature?

Reply:
Yes, I moved the function definition next to the other static functions.